meson.build

```diff
@@ -63,7 +63,7 @@ csp_inc = include_directories(['.', 'include'])
 install_subdir('include', install_dir : '.')
 install_headers(csp_config_h, install_dir : 'include/csp')
 
-csp_deps += dependency('freertos', fallback: ['freertos', 'freertos_dep'], required: false)
+csp_deps += dependency('freertos', required: false)
```

Comment:
Why would you remove the fallback? That requires FreeRTOS to be installed in the system path, and be found by the libtool, which I think is quite rare. The fallback enabled the possibility to have FreeRTOS as another meson subproject.

Reply:
It compiles just fine with my version of meson (0.54.2) with the subprojects using this syntax.
It didn't work if you don't have an explicit FreeRTOS dependency however, breaking compatibility with our Zynq project (that has FreeRTOS hidden under another dependency and not explicitly providing a FreeRTOS dependency).

Reply:
1. According to the meson docs, It shouldn't be working without the fallback. The reason why your build does work, must be because FreeRTOS is found somewhere in the search path.

2. It is against the meson philosophy to hide libraries in other libraries. But yes, that is how it is done within the zynq BSP. I think we somewhere have a dependency on either freertos or bsp in order to include the headers from the BSP instead, as a workaround. 

But anyway, I cannot see why removing the fallback should fix any of that.?